controllers/dashboard_test.go

```diff
@@ -34,6 +34,6 @@ var _ = Describe("DashboardController", func() {
 	})
 
 	AfterEach(func() {
-		TruncateTable("users")
+		TruncateTables("users")
```

Comment:
I changed [this](https://github.com/junan/google-scraper/pull/30/files#diff-effa542a732549b80b054f420e6165edfb646dcb3e2599d1eb5a4f8571f5dfc7R10) function, now it can [take](https://github.com/junan/google-scraper/pull/30/files#diff-864a04f616d07c494c73921e767a8d80d4e74d276398108538f0f48a0eebafd1R132) many tables